src/actions/PermissionsActions.js

```diff
@@ -55,13 +55,28 @@ export const promptForLocationServicesPermission = () => {
     };
 };
 
-const getImageFromGallery = async (dispatch) => {
-    let result = await ImagePicker.launchImageLibraryAsync({
-        allowsEditing: true,
-        aspect: [4, 4],
-        base64: true,
-        quality: 2
-    });
+const getImage = async (dispatch, imageSourceType) => {
+    let result = {cancelled: true};
+
+    if (imageSourceType === imageSourceTypes.CAMERA_ROLL) {
+        console.log("In the CAMERA_ROLL area: ",imageSourceType);
+
+        result = await ImagePicker.launchImageLibraryAsync({
+            allowsEditing: true,
+            // aspect: [4, 4],
+            base64: true,
+            quality: 0.5
+        });
+    } else if (imageSourceType === imageSourceTypes.CAMERA) {
+        console.log("In the CAMERA area: ",imageSourceType);
+
+        result = await ImagePicker.launchCameraAsync({
+            allowsEditing: true,
+            // aspect: [4, 4],
+            base64: true,
+            quality: 0.5
+        });
+    }
```

Comment:
Made `getImageFromGallery` into a more generic `getImage` which takes an argument of `imageSourceType` to see if it should try to get the image from the `CAMERA` or `CAMERA_ROLL`